src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -196,4 +193,23 @@ public ResponseEntity<Void> deleteLineTypesCatalog() {
         lineTypesCatalogService.deleteLineTypesCatalog();
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping(value = "/groups/modification", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a group containing a modification")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The group with the modification has been created")})
+    public ResponseEntity<UUID> createModificationInGroup(@RequestBody ModificationInfos modificationsInfos) {
```

Comment:
Why not adding a entrypoint to create a `groups` resource, then use the existing endpoint to create a `network-modifications` resource ?

Reply:
I see, there isn't group resource, it's only an UUID, my bad !